docs/guide/resources.md

```diff
@@ -19,6 +19,15 @@
 | [Biome's noReExportAll](https://biomejs.dev/linter/rules/no-re-export-all/) | Biome plugin to avoid re-export all |
 | [unplugin-purge-polyfills](https://github.com/danielroe/unplugin-purge-polyfills) | Unplugin to replace package imports with native code |
 
+## Websites
+
+|  | Description |
+| -- | -- |
+| [DepTree](https://deptree.rschristian.dev/) | Visualizes a projects dependencies and highlights which of those dependencies could be replaced |
+| [npmgraph](https://npmgraph.js.org/) | Visually displays a graph of a project's dependencies to get an idea of a projects module graph |
+| [Ecosyste.ms](https://packages.ecosyste.ms/) | An open API service providing package, version and dependency metadata of many open source software ecosystems and registries |
+
 ## Learning
 
 - [Marvin's Speeding Up the JS Ecosystem series](https://marvinh.dev/blog/speeding-up-javascript-ecosystem/)
+- [Practical barrel file guide for library authors](https://thepassle.netlify.app/blog/practical-barrel-file-guide-for-library-authors)
```

Comment:
Not sure why this is showing as added 🤔 it's already in the `main` branch

<img width="486" alt="image" src="https://github.com/user-attachments/assets/26529872-8902-4b63-a618-f82ad9b0373d">

Reply:
Probably because of the new line ending being affected